### ISSUE TITLE
There is no tag for 0.4.2 version

### ISSUE THREAD

**Issue by uilianries** [tartley/colorama#241]
The version 0.4.2 has been released in Pypi:

https://pypi.org/project/colorama/0.4.2/

However, there is no tag to that version here in Github.

**Comment by dgjustice** [comment]
Looks like it's broken, too.
```
Collecting colorama
  Using cached https://files.pythonhosted.org/packages/cb/fe/bfc4d807aa43a183ab387340f524a0bb086624f2c5935bd08e647b54b269/colorama-0.4.2.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-wvqpxpre/colorama/setup.py'"'"'; __file__='"'"'/tmp/pip-install-wvqpxpre/colorama/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-wvqpxpre/colorama/pip-egg-info
         cwd: /tmp/pip-install-wvqpxpre/colorama/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-wvqpxpre/colorama/setup.py", line 36, in <module>
        long_description=read_file('README.rst'),
      File "/tmp/pip-install-wvqpxpre/colorama/setup.py", line 18, in read_file
        with open(os.path.join(os.path.dirname(__file__), path)) as fp:
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-install-wvqpxpre/colorama/README.rst'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by uilianries** [comment]
@dgjustice Please, upvote https://github.com/tartley/colorama/issues/240

**Comment by tartley** [comment]
Apologies. Should be fixed now with the release of 0.4.3.

I'm closing this for now. Any outstanding problems or diagnostic info can go into #240 .

Thank you both!

**Comment by uilianries** [comment]
Thanks!
